docs/indices.rst

```diff
@@ -23,6 +23,13 @@ Glossary
        A community Python library for Astronomy:
        https://www.astropy.org/.
 
+   bokeh
+       The bokeh plotting package, which is documented at
+       http://bokeh.org/, is used to provide the plotting support
```

Comment:
"used to provide **the** plotting support in sherpa" should be changed "used to provide plotting support in sherpa" as the word 'the' changes the statement's connotation.

Reply:
I'll address that in a local change, because that means I should also modify the matplotlib glossary entry to make them match.